geode-core/src/main/java/org/apache/geode/internal/cache/backup/BackupService.java

```diff
@@ -74,6 +74,9 @@ public BackupService(InternalCache cache) {
     try {
       result = taskFuture.get();
     } catch (InterruptedException | ExecutionException e) {
+      if (e instanceof ExecutionException) {
```

Comment:
I think that it will be good to log both exceptions.
Please remove this check.

Reply:
Done. Thanks!